### ISSUE TITLE
Cannot pickup

### ISSUE THREAD

**Issue by zxyao145** [casesandberg/react-color#741]
I cannot select a color, even if it is in a new react project. 

version:

>
> "react": "^16.13.1",
> "react-color": "^2.18.1",
> 

![bug](https://user-images.githubusercontent.com/24973084/88280454-6b389180-cd18-11ea-8bab-dfdf809b7e93.gif)

**Comment by rakibpaucse** [comment]
Did you use onChangeComplete ?

**Comment by zxyao145** [comment]
@rakibpaucse 

Yes, and I can see the output on the console, but I can't use the mouse to manipulate it.

**Comment by zxyao145** [comment]
@rakibpaucse 

I know what happened. I have to update the color myself, which is not the state of react-color, but the props.

Thanks a lot!

**Comment by utiq** [comment]
@zxyao145 @rakibpaucse I'm running through the same problem, I'm using functional components, can you share your code to see how you did it?

**Comment by zxyao145** [comment]
@utiq 
You hava to have a state to storage the color, then pass it as an argument to the sketchpicker's color props, and update the state on the event `onChangeComplete` or `onChange`. 

I'm using the class components, and if you want to use functional components, you need to use the hook function.

My code snippet:

```jsx

this.state = {
      background: props.color || { r: 10, g: 10, b: 90, a: 50 },
};

handleChangeComplete(color) {
    if (this.props.onColorChanged) {
      this.props.onColorChanged(color.rgb);
    }
  }

handleChange(color, event) {
    this.setState({ background: color.rgb });
}

<SketchPicker
        className="color-pick"
        color={this.state.background}
        onChangeComplete={this.handleChangeComplete.bind(this)}
        onChange={this.handleChange.bind(this)}
 />
```
